charmhelpers/contrib/openstack/context.py

```diff
@@ -500,6 +500,7 @@ def _resolve(key):
                 int_host = format_ipv6_addr(int_host) or int_host
                 svc_protocol = _resolve('service_protocol') or 'http'
                 auth_protocol = _resolve('auth_protocol') or 'http'
+                admin_role = _resolve('admin_role') or 'Admin'
```

Comment:
I don't see a test to handle the `_resolve('admin_role')` condition here. Is this a possibility to add?

Reply:
Added test for this new property